OxfordBodleian/Juel-Jensen/BDLaethg41.xml

```diff
@@ -50,35 +152,176 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                     <material key="parchment"/>
                                 </support>
                                 <extent>
-                                    <!--<measure unit=""></measure>-->
+                                    <note>Consists of <measure unit="strips">2</measure> strips sewn together.</note>
                                     <dimensions type="outer" unit="mm">
                                         <height>1648</height>
-                                        <width>66-77</width>
+                                        <width>66–77</width>
+                                    </dimensions>
+                                    <dimensions type="outer" unit="mm" xml:id="strip1">
+                                        <height>865</height>
+                                        <width>66–76</width>
+                                    </dimensions>
+                                    <dimensions type="outer" unit="mm" xml:id="strip2">
+                                        <height>805</height>
+                                        <width>68–77</width>
                                     </dimensions>
                                     <measure type="weight" unit="g">28</measure>
                                 </extent>
+                                <condition key="good">The state of preservation is good. One small hole in the upper part of the scroll. The lower part of the scroll has been crumpled by rolling,
```

Comment:
instead of 'small hole' which could be misunderstood with a hole pierced in the scroll I propose 'small areas of loss'. Looking at the back it seems that also the tiny strip of the upper margin missing. What do you think?

Reply:
Yes, the top is not so well photographed (to our excuse I can say that it is very difficult with stiff scrolls...)....Small areas of loss is good, I think